src/proposals/proposal.cpp

```diff
@@ -151,9 +151,10 @@ namespace hypha
     bool Proposal::didPass(const eosio::checksum256 &tallyHash)
     {
         TRACE_FUNCTION()
-        name hvoiceContract = m_dao.getSettingOrFail<eosio::name>(HVOICE_TOKEN_CONTRACT);
-        hypha::voice::stats stats_t(hvoiceContract, common::S_HVOICE.code().raw());
-        auto stat_itr = stats_t.find(common::S_HVOICE.code().raw());
+        name hvoiceContract = m_dao.getSettingOrFail<eosio::name>(GOVERNANCE_TOKEN_CONTRACT);
+
+        hypha::voice::stats stats_t(hvoiceContract, common::S_VOICE.code().raw());
+        auto stat_itr = stats_t.find(common::S_VOICE.code().raw());
         EOS_CHECK(stat_itr != stats_t.end(), "No HVOICE found");
 
         asset quorum_threshold = adjustAsset(stat_itr->supply, 0.20000000);
```

Comment:
This is where the 20% quorum and 80% alignment calculations take place.  We need to move these values to the DHO's settings document.